RELEASE/scripts/autoscend/iotms/mr2021.ash

```diff
@@ -114,10 +114,10 @@ void simulatePreAdvForCrystalBall(location place)
 	// used only when simulating maximizer equipment
 	// replicates most of pre_adv monster queue checks in order to know if miniature crystal ball will be allowed
 	
-	boolean burningDelay = ((auto_voteMonster(true) || isOverdueDigitize() || auto_sausageGoblin() || auto_backupTarget()) && place == solveDelayZone());
-	boolean gettingLucky = (have_effect($effect[Lucky!]) > 0 && zone_hasLuckyAdventure(place));
+	boolean burningDelay = auto_burningDelay();
+	boolean gettingLucky = auto_gettingLucky();
 	boolean forcedNonCombat = auto_haveQueuedForcedNonCombat();
-	boolean zoneQueueIgnored = (burningDelay || gettingLucky || forcedNonCombat);
+	boolean zoneQueueIgnored = auto_queueIgnore();

```

Comment:
this one ||s the three computed above and probably doesn't need to be a function that computes all three again